### DISCUSSION TITLE
Access to build history

### DISCUSSION THREAD

**Discussion by lukaszkazmierczak** [actions/virtual-environments#10539] [General questions] [answered]
Hello,

I cannot access to the build history (https://github.visualstudio.com/runner-images/_build) at this moment (404 error). In the past I've used the page to compare my build, errors or analysing failures. Does the URL has been changed or the public access has been blocked?

Thanks,
Lukasz

**Comment by erik-bershel** [comment, marked as answer]
Hey @lukaszkazmierczak!

For some internal reason we had to get rid of the specified repository for testing public builds. We'll announce the new method once it will become available.
